### ISSUE TITLE
Show feedback for Hardware Security Key

### ISSUE THREAD

**Issue by cassidyjames** [elementary/greeter#393]
If you require a FIDO U2F key in PAM for LightDM, the greeter currently correctly waits a moment after submitting your password to let you authenticate with the key. However, there's no on-screen feedback.

Instead, we should show an icon or some sort of indication when a hardware key is required, and show visual feedback while waiting for authentication from the hardware key.

**Comment by danrabbit** [comment]
Yeah I would picture this something like the numlock indicator where we'd have a bit of messaging that reminds you if your key is not plugged in

**Comment by cassidyjames** [comment]
@danrabbit yeah makes sense. After entering a password, we might also need a spinner with messaging that is like, "Tap your security key" or something; I'll try to see how Windows and web apps that support U2F keys say it. But it turns out having the key inserted is not enough; you need to tap (usually a metal contact of some sort) on the key so that it can't passively authenticate.
